### ISSUE TITLE
Add onboarding checklist

### ISSUE THREAD

**Issue by ericholscher** [readthedocs/ext-theme#438]

**Comment by humitos** [comment]
Should we try to prioritize this? I think it could make the onboarding more pleasant. We already have all the data in the DB. We only need to show it as an announcement in the right bar using what we already have.

**Comment by ericholscher** [comment]
Seems like a reasonable thing to test out -- though we'd probably want to think about how the UX would work so it wasn't annoying.

**Comment by humitos** [comment]
I'm not sure what you have in mind here. My thoughts are:

1. show a "closable" announcement similar to the other ones we have. The user can close it at any time and it won't be shown anymore
2. only show it if there are items still left
3. each item could have a link to the docs
4. mark the items as done when the user enable the feature

I will see if I can create a draft with AI to discuss about it.

**Comment by humitos** [comment]
This is what I got,

<img width="385" height="554" alt="Image" src="https://github.com/user-attachments/assets/04a9bba6-a4c3-4f2a-89e6-c679ff8c6e3c" />

it's similar to what I had in mind, but I don't like the style about how it looks. I tried to use some fomatic-sui CSS classes to improve it but I didn't find anything that I liked.

**Comment by ericholscher** [comment]
Yeah, I think the main issue is just making sure it gets closed at a reasonable point in time. Having onboarding stuff stick around and getting in the way is pretty annoying. 

I actually don't care too much about what it looks like, but really want to think about how it works, when we show it, and especially when it gets removed automatically.
